hail/src/main/scala/is/hail/services/Requester.scala

```diff
@@ -19,7 +23,52 @@ import org.apache.log4j.{LogManager, Logger}
 import org.json4s.{DefaultFormats, Formats, JObject, JValue}
 import org.json4s.jackson.JsonMethods
 
+import scala.collection.JavaConverters._
 import scala.util.Random
+import java.io.FileInputStream
+
+
+abstract class CloudCredentials {
```

Comment:
this name is discordant with the name you used in Python for a thing that returns access tokens.

Reply:
I've now gotten rid of the `Credentials` vs `CloudCredentials` in python and just brought it back to `CloudCredentials` so hopefully this is aligns better now.